spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -548,6 +555,34 @@ def then_I_am_told_that_it_is_published
     expect(page).to have_content("has been published")
   end
 
+  def and_I_have_reordered_the_steps
+    publishing_api_receives_get_content_id_request(
+      content_items: step_link_fixtures_content_items,
+    )
+    visit step_by_step_page_reorder_path(@step_by_step_page)
+    find(".js-reorder:first-child .js-down").click
+    click_button "Save"
+    expect(page).to have_content "Steps were successfully reordered."
+    and_I_should_be_on_the_step_by_step_page
+  end
+
+  def when_I_add_another_step
```

Comment:
Could this be renamed `when_I_add_a_step` since it isn't dependent on there already being steps?